SteamKit2/SteamKit2/Base/ClientMsg.cs

```diff
@@ -28,14 +28,15 @@ internal ClientMsgProtobuf( int payloadReserve = 0 )
         /// <value>
         /// 	<c>true</c> if this instance is protobuf backed; otherwise, <c>false</c>.
         /// </value>
-        public override bool IsProto { get { return true; } }
+        public override bool IsProto => true;
+
         /// <summary>
         /// Gets the network message type of this client message.
         /// </summary>
         /// <value>
         /// The network message type.
         /// </value>
-        public override EMsg MsgType { get { return Header.Msg; } }
+        public override EMsg MsgType => Header.Msg;

```

Comment:
good to know then

On Sat, Nov 7, 2015 at 4:53 AM, Netshroud <notifications@github.com> wrote:

> In SteamKit2/SteamKit2/Base/ClientMsg.cs
> <https://github.com/SteamRE/SteamKit/pull/198#discussion_r44210501>:
>
> >          /// <summary>
> >          /// Gets the network message type of this client message.
> >          /// </summary>
> >          /// <value>
> >          /// The network message type.
> >          /// </value>
> > -        public override EMsg MsgType { get { return Header.Msg; } }
> > +        public override EMsg MsgType => Header.Msg;
>
> This is patently false.
>
> { get; } = ... instructs the compiler to emit an assignment to the
> backing field in the class's constructor (or static constructor for a
> static property).
>
> DefaultValueAttribute does nothing, a framework has to explicitly search
> for it for it to be of use.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SteamRE/SteamKit/pull/198/files#r44210501>.
>